src/components/login.component.js

```diff
@@ -1,23 +1,81 @@
 import React from "react";
 import "./login.component.css";
+import axios from "axios";
+import * as Constants from "../constants";
```

Comment:
It may be worth importing `SIGN_IN_USER_API` explicitly if we only need one value from the constants file.